generators/component/index.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+var yeoman = require('yeoman-generator');
+var _ = require('lodash');
+
+module.exports = yeoman.Base.extend({
+  constructor: function() {
+    yeoman.Base.apply(this, arguments);
+
+    this.argument('componentName', {
+      type: String,
+      required: true,
+      description: 'The name of the component'
+    });
+  },
+
+  writing: function () {
+    var componentDir = "client/app/components/"+_.kebabCase(componentName);
+    var templateVars = {
+      _,
+      componentName: this.componentName
+    };
+
+    this.fs.copyTpl(
+      this.templatePath('component.js'),
+      this.destinationPath(componentDir+"/"+_.kebabCase(componentName)+".js"),
+      templateVars
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('component.css'),
+      this.destinationPath(componentDir+"/"+_.kebabCase(componentName)+".css"),
+      templateVars
+    );
+
+    this.fs.copyTpl(
+      this.templatePath('index.js'),
+      this.destinationPath(componentDir+"/"+"index.js"),
```

Comment:
But this needs to generate in some structure, I think that the generator is useful when you have to build all those files by hand. To create just a new file it's easier to create by hand.

But maybe prompt the user(or an option to get out of the way) is the way to go